contributor-growth/README.md

```diff
@@ -0,0 +1,65 @@
+# Contributor Growth
+
+The Contributor Growth subproject is interested in assisting CNCF projects with
+sustainably growing their contributor base.
+
+## Goals
+
+* Provide best practices and resources for managing and interacting with
+  contributors to CNCF projects.
+* Provide guidance on building a contributor pipeline that brings in new
+  contributors and promotes them to higher levels of responsibility within CNCF
+  projects, including the maintainer role.
+
+## Scope
+
+* Templates for common documents like a contributing guide, reviewing guide,
+  maintainers guide, contribution ladder, etc.
+* Case studies of sucessful projects of various kinds/sizes and how they manage
+  contributions.
+* Collect and share talks related to the subproject's mission, for example
+  building open source communities, or reflections on a particular community's
+  best practices.
+* Document ways to contribute to a project, especially roles outside of the
+  obvious feature enhancement/code contributions.
+* Create spaces for maintainers of CNCF projects to share their own best
+  practices and ask questions.
+* Discuss non-code areas that support contributor growth, such as project
+  management, design, user experience and documentation and how CNCF projects
+  can prioritize work that may not be the maintainers primary skillset.
+* Provide materials that a CNCF project can use to address the TOC [due
+  dilligence guidelines] related to project's contributors and community.
+
+[due dilligence guidelines]: https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md#project
+
+### Out-of-Scope
+
+The following are exmaples of things out-of-scope:
+
+* Project governance.
+* Project automation, such as prow.
+* Day to day support for improving the contribution experience for CNCF
+  projects. That falls under other projects, for example Kubernetes SIG
```

Comment:
Can we clarify what "other projects" mean? Do we mean within a CNCF project's own scope, with the example being K8s SIG Contributor Experience?